### DISCUSSION TITLE
Method Filament\Tables\Columns\TextColumn::expandableLimitedList does not exist.

### DISCUSSION THREAD

**Discussion by NaungYeHtet** [filamentphp/filament#9991] [Help] [answered]
I have Json column numbers casted to array in Model

```
Tables\Columns\TextColumn::make('numbers')
    ->listWithLineBreaks()
    ->limitList(5)
    ->expandableLimitedList()
```

**Comment by leandrocfe** [comment, marked as answer]
composer update?
What is the Filament version on the dashboard page?

**Reply by NaungYeHtet** [reply]
composer update works, thanks a lot.

**Comment by SayidMuhammad007** [comment]
Method Filament\Tables\Columns\TextColumn::verticalAlignment does not exist.
how can i fix this problem?
